core/transformer/src/IModelExporter.ts

```diff
@@ -137,6 +137,8 @@ export class IModelExporter {
   public readonly sourceDb: IModelDb;
   /** A flag that indicates whether element GeometryStreams are loaded or not.
    * @note As an optimization, exporters that don't need geometry can set this flag to `false`. The default is `true`.
+   * @note the transformer by default sets this to false as an optimization
```

Comment:
Do you know if the doc generator collapses these into one note, or produces three separate notes? Which were you aiming for?

Reply:
looks like we get three separate notes: https://www.itwinjs.org/reference/core-transformer/imodels/imodeltransformoptions/preserveelementidsforfiltering/

Reply:
I'm ok with that format. Paragraph format would be good to. I'll change it to paragraph format if you prefer

Reply:
Up to you. Either way prefer to use proper sentence capitalization and punctuation.

Reply:
Will keep multiple notes but fix sentences

Reply:
done for this and a few other doc comments